recipes-support/salt/files/set_python_location_hashbang.patch

```diff
@@ -0,0 +1,107 @@
+Upstream-Status: Pending
+
+# The Salt SysV scripts require that the process name of the salt
+# components have the form "salt-<component>".
+# The current python shebangs on the salt components scripts spwans
+# processes that are generically named python. Changed shebang so
+# process names will be identifiable by the init scripts.
```

Comment:
Since we own the sysVinit scripts, can we just fixup the initscript's logic to correctly handle the salt python scripts? The shebang lines they're using is best-practice, and I would prefer a local modification to our initscript over applying a `.patch` to a dozen files.

Reply:
I'm not sure how we can match a python process in an init script, if not for the process name.
Also, I think a change like this would generate more churn than necessary at this time.

Reply:
I'm pretty sure `start-stop-daemon` could still match the process based on the `PIDFILE` alone, and the `--name` argument is just there to catch the case that the original process was killed and another process happened to startup with the same PID. That being said, I understand the complication with running a python script which uses `env` as an indirection layer and it seeting `/proc/$pid/comm` to `python` instead of the script name.

I still think there is a better way to handle this, but after searching it isn't obvious and I'm not willing to block these changes on it.